packages-exp/auth-exp/src/core/auth/auth_impl.ts

```diff
@@ -267,7 +260,8 @@ export class AuthImpl implements AuthInternal, _FirebaseService {
         true
       );
     } catch (e) {
-      this.redirectInitializerError = e;
+      // Swallow any errors here; the code can retrieve them in
+      // getRedirectResult().
       await this._setRedirectUser(null);
```

Comment:
do you have a test for the catch?

Reply:
Ya there is a test for this code but the error was never caught since the test uses the initialization function that leaves the promise hanging. I've updated this test so that it would catch an error being thrown :+1:

Reply:
ty!